### DISCUSSION TITLE
Are you accepting UX design pull requests?

### DISCUSSION THREAD

**Discussion by alancmclean** [dvmarinoff/Auuki#172] [General]
Hi there. First, I want to say how grateful I am for seeing this amazing project. I got sick of paying Zwift and Trainerroad of basic functionality and contemplated doing something like this for myself. It’s great to see something being built in the open.

I’m wondering if you’re accepting pull requests that are UX centric. I don’t really have the technical skills to make some of changes I’ve been thinking about. However I have been working as a designer for these kinds of things for a while at places like Strava, Fitbit, Google among others. If you are open to it I can provide designs in Figma. 

If not, no big deal. I’m happy to continue with it as is and appreciate your efforts.

**Comment by dvmarinoff** [comment]
@alancmclean Hey, thanks, sure I’d be happy to take some input on the UX side!

**Reply by sm3llmys0x** [reply]
@alancmclean have you given any feedback on UX yet? Curious, because although I love this app and think it's pretty damn cool, I can't say that the interface is very intuitive. I was going to start playing around with the interface to see what I could come up with.

**Reply by alshdavid** [reply]
I'd be happy to write a UI to accommodate UX improvements but I'll need help wiring up the WebBT/etc APIs and charting system you've implemented. 

I see a lot of requests for self hosting. How is the app deployed and how does authentication work (I can't see a backend in the code, is auth client-side with a client-side authentication provider)? I can add a pipelines workflow to deploy to github pages so people can self host by simply forking the project.

**Reply by dvmarinoff** [reply]
just started this document: https://github.com/dvmarinoff/Auuki/discussions/281, will keep adding to it over time.
